scipy/stats/stats.py

```diff
@@ -1037,8 +1040,32 @@ def moment(a, moment=1, axis=0):
        is None. The denominator for the moment calculation is the number of
        observations, no degrees of freedom correction is done.
 
+    See also
+    --------
+    kurtosis, skew, describe
+
+    Notes
+    -----
+    [1]_ The n-th moment of a real-valued continuous function f(x) of a real
+    variable about a value c is:
+        \mu _{n} = \int_{-\infty }^{\infty }(x - c)^n  f(x) dx
```

Comment:
I agree. So the formula should be a finite sum instead of an infinite integral. Here's a formula: http://www.r-tutor.com/elementary-statistics/numerical-measures/moment